partcad-cli/src/partcad_cli/click/commands/list/assemblies.py

```
@@ -21,21 +21,27 @@
 
         if used_by is not None:
-            logging.info(f"Instantiating {used_by}...")
-            # TODO(clairbee): do not call it twice in 'list all'
-            ctx.get_assembly(used_by)
+            pc_logging.info(f"Instantiating {used_by}...")
+            try:
+                target = ctx.get_assembly(used_by)
+                if not target:
+                    pc_logging.error(f"Assembly {used_by} is not found")
+                    return
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)